### PR TITLE
Attached volumes to postgres, hbase nad rabbitmq containers

### PR DESCRIPTION
**Pull request by kolszewska**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -2,6 +2,8 @@ version: '3'
 services:
   hbase:
     image: dajobe/hbase
+    volumes:
+      - hbase-data:/data
```
Comment (jpowie01) [inline docker-compose.yml]: I would use `.medtagger/hbase-data` and please add `.medtagger` to .gitignore.

File: docker-compose.yml [hunk 2]
```diff
@@ -12,6 +14,8 @@ services:
 
   rabbitmq:
     image: rabbitmq:management
+    volumes:
+      - rabbitmq-data:/var/lib/rabbitmq/mnesia
```
Comment (jpowie01) [inline docker-compose.yml]: I would use `.medtagger/rabbitmq-data` and please add `.medtagger` to .gitignore.

File: docker-compose.yml [hunk 3]
```diff
@@ -24,6 +28,8 @@ services:
     image: postgres
     ports:
       - 5432:5432
+    volumes:
+      - postgres-data:/var/lib/postgresql/data
```
Comment (jpowie01) [inline docker-compose.yml]: I would use `.medtagger/postgres-data` and please add `.medtagger` to .gitignore.

**Comment by kolszewska** [comment]
Fixes #35

**Pull request merged**